### PR TITLE
Add Blazor eBook TOC

### PR DESCRIPTION
**Pull request by danroth27**

### CODE REVIEW

File: docs/architecture/blazor-for-web-forms-developers/pages-and-routing.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (mairaw) [inline docs/architecture/blazor-for-web-forms-developers/pages-and-routing.md]: pages-and-routing.md -> pages-routing.md

We don't use small unnecessary words: https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=master#name-your-files

File: docs/architecture/blazor-for-web-forms-developers/security.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (mairaw) [inline docs/architecture/blazor-for-web-forms-developers/security.md]: security.md -> security-authentication-authorization.md

File: docs/architecture/blazor-for-web-forms-developers/startup.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (mairaw) [inline docs/architecture/blazor-for-web-forms-developers/startup.md]: startup.md -> app-startup.md

**Review by twsouthwick — APPROVED** [review]

**Comment by danroth27** [comment]
@mairaw :up::date:

**Review by mairaw — APPROVED** [review]
LGTM. My last commit should fix the broken links warnings.

**Pull request merged**